src/otx/recipe/classification/multi_label_cls/mobilenet_v3_large.yaml

```diff
@@ -1,8 +1,9 @@
 model:
-  class_path: otx.algo.classification.mobilenet_v3.MobileNetV3ForMultilabelCls
+  class_path: otx.algo.classification.mobilenet_v3.MobileNetV3ForClassification
   init_args:
     mode: large
     label_info: 1000
+    task: MULTI_LABEL_CLS
```

Comment:
Just a question, don't we need to use `task` or `train_type` as global configurations?

Reply:
For now, the current refactoring is to separate the 3 sub-tasks of Classification. Actually, I think this could be added to the `OTXModel` more globally as you said, but the other tasks don't need to be given a choice because each model is currently only available for a specific task.
@kprokofi I think we can take that as a discussion topic as well.

Reply:
Yes, we need to discuss it.
I think `task` for detection can be divided into `detection`, `rotated detection`, and `keypoint detection`, and `task` for visual prompting can be divided into `fine tuning` and `zero-shot learning`.
Also, `train_type` can be used across all tasks like semi-supervised learning or self-supervised learning for semantic segmentation.